package-lock.json

```diff
@@ -6,7 +6,7 @@
   "packages": {
     "": {
       "name": "youtube-webos",
-      "version": "0.3.3",
+      "version": "0.3.4",
```

Comment:
Please remove this.

Reply:
Somehow got added automatically - removed.